docs/reference/ccr/overview.asciidoc

```diff
@@ -17,7 +17,7 @@ follower index. This simplifies state management on the leader index and means
 that {ccr} does not interfere with indexing on the leader index.
 
 [float]
-=== Configuring replication
+=== Initiating replication
```

Comment:
I'm curious about the intended change in meaning here. Is it meant to cover both configuring and starting CCR?

Reply:
I guess so. The actual `create follower` and `auto follower` actions seem more associated with initiating replication (although you can configure certain things using parameters). That is why I made that change. Thoughts?

Reply:
I find "initiating" a bit unclear, so my vote would be simpler verbs like "configuring" or "setting up", but it's not a deal breaker.

Reply:
I changed it back to `configuring`